### PR TITLE
Saving configuration to localStorage

### PR DESCRIPTION
**Pull request by cobicarmel**

### CODE REVIEW

File: includes/pojo-a11y-settings.php [hunk 1]
```diff
@@ -302,9 +302,37 @@ public function section_a11y_settings( $sections ) {
...
+				48 => __( '2 Days', 'pojo-accessibility' ),
+				72 => __( '3 Days', 'pojo-accessibility' ),
+				168 => __( '1 Week', 'pojo-accessibility' ),
+				720 => __( '1 Month', 'pojo-accessibility' ),
+			),
+			'std' => 12,
+		);
 		
 		$sections[] = array(
-			'id' => 'section-a11y-settings',
```
Comment (KingYes) [inline includes/pojo-a11y-settings.php]: Why you change this section ID?

Comment (cobicarmel) [inline includes/pojo-a11y-settings.php]: By mistake. I'll send a fixing right now

File: assets/js/app.dev.js [hunk 2]
```diff
@@ -51,91 +67,183 @@
...
-				if ( 'plus' === action && MAX_SIZE > oldFontSize ) {
-					$self.currentFontSize += 10;
+				if ( 'reset' === action ) {
+					Pojo_Accessibility_App.reset();
+					return;
 				}
 
-				if ( 'minus' === action && MIN_SIZE < oldFontSize ) {
-					$self.currentFontSize -= 10;
+				if ( [ 'toggle', 'schema' ].indexOf( actionGroup ) + 1 ) {
```
Comment (KingYes) [inline assets/js/app.dev.js]: Please use `( -1 === [ 'toggle', 'schema' ].indexOf( actionGroup ) )`.

File: assets/js/app.dev.js [hunk 3]
```diff
@@ -51,91 +67,183 @@
...
-		
-		bindToolbarButtons: function() {
-			var $self = this;
 
-			$self.currentFontSize = 120;
+		bindToolbarButtons: function() {
+			var self = this;
 
-			$self.cache.$toolbar.find( 'a.pojo-a11y-btn-resize-font' ).on( 'click', function( event ) {
+			this.cache.$toolbarLinks.on( 'click', function( event ) {
```
Comment (KingYes) [inline assets/js/app.dev.js]: You have `self` var for that

File: assets/js/app.dev.js [hunk 4]
```diff
@@ -51,91 +67,183 @@
...
-					oldFontSize = $self.currentFontSize;
+				var $this = $( this ),
+					action = $this.data( 'action' ),
+					actionGroup = $this.data( 'action-group' ),
+					deactivate = false;
 
-				if ( 'plus' === action && MAX_SIZE > oldFontSize ) {
-					$self.currentFontSize += 10;
+				if ( 'reset' === action ) {
+					Pojo_Accessibility_App.reset();
```
Comment (KingYes) [inline assets/js/app.dev.js]: Why you don't write `self.reset();` ?

**Pull request merged**